src/components/lco/PageBanner.tsx

```diff
@@ -124,6 +136,100 @@ interface ContentProps {
 //  )
 // }
 
+const LcoCard: React.FC<ContentProps> = ({ profile }) => {
+  const { name, website, instagram, report, donation } = profile
+  return (
+
+    <div className='grid md:grid-cols-6 px-11 pt-11 pb-16 gap-2 md:gap-4 bg-white rounded-lg'>
+
+      <div className='md:col-span-1'>
+        <UsersIcon className='h-12 w-12 rounded-lg border-slate-100 border' />
+      </div>
+
+      <div className='gap-10 md:col-span-5'>
+        <p className='text-base-content/60 font-semibold'>Local Climbing Organization</p>
+        <h2 className='card-title my-2 uppercase'>{name}</h2>
+        <a
+          className='underline pb-5'
+          href={website}
+          target='_blank'
+          rel='noreferrer'
+        >
+          <p className='text-sm'>{website}</p>
+        </a>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure
+          reiciendis, illo alias blanditiis corporis temporibus a. Sit animi
+          accusamus laborum minima voluptatibus perspiciatis quos vitae,
+          temporibus, ea quam fugit eos.
+        </p>
+        <div className='border-t border-b divide-y'>
+          <div className='flex felx-row flex-wrap justify-between pt-4 md:pt-6'>
+            <a
+              className='underline pb-4'
+              href={instagram}
+              target='_blank'
+              rel='noreferrer'
+            >
+              contact info
+            </a>
+            <a
+              className='underline pb-4'
+              href={instagram}
+              target='_blank'
+              rel='noreferrer'
+            >
+              instagram
+            </a>
+            <a
+              className='underline pb-4'
+              href={instagram}
+              target='_blank'
+              rel='noreferrer'
+            >
+              twitter
+            </a>
+            <a
+              className='underline pb-4'
+              href={instagram}
+              target='_blank'
+              rel='noreferrer'
+            >
+              facebook
+            </a>
+          </div>
+        </div>
+        <div className='space-y-4 py-6'>
+          <div>
+            {donation != null && (
+              <a
+                className='btn btn-primary btn-outline'
+                href={donation}
+                target='_blank'
+                rel='noreferrer'
+              >
+                Make a Donation <ArrowUpRightIcon className='ml-2 w-4 h-4' />
+              </a>
+            )}
+          </div>
+          <div className='card-actions '>
+            <a
+              className='btn btn-primary opacity-80 btn-solid bg-red-500 border-0'
```

Comment:
For the primary CTA should we just use `btn-solid` (dark black) instead of introducing a new custom color?

Reply:
Yeah sure! I'll change it.